tests/common/helpers.py

```diff
@@ -230,3 +230,11 @@ def _next_timestamp():
             raise RuntimeError("Is DateTime.now() frozen ?")
 
     return _next_timestamp
+
+
+import structlog
+
+
+@pytest.fixture
+def log() -> structlog.BoundLogger:
```

Comment:
What is this for ?

Just use print in test instead (pytest takes care of capturing stdout when the test doesn't run with --verbose)

Reply:
done